protocol/bin/flow.js

```diff
@@ -1,20 +1,19 @@
-'use strict'
+'use strict' // eslint-disable-line
```

Comment:
just some general cleanup of this file. more arrows and es6 stuff that node handles out of the box